src/EditorFeatures/Core/Classification/Semantic/ClassificationUtilities.cs

```diff
@@ -174,7 +174,7 @@ public static SegmentedList<ITagSpan<IClassificationTag>> Convert(IClassificatio
                     var version = await document.Project.GetDependentSemanticVersionAsync(cancellationToken).ConfigureAwait(false);
 
                     // Let the context know that this was the span we actually tried to tag.
-                    context.SetSpansTagged(ImmutableArray.Create(snapshotSpan));
+                    context.AddActuallyTaggedSpan(snapshotSpan);
```

Comment:
this was the crux of teh bug.  overwriting vs accumulating.  in the past this really didn't matter as there was no tagger that actually produced multiple disparate spans and also called SetSpansTagged.  So we never noticed this issue.